### PR TITLE
Prepare Core for January 2021 Release

### PR DESCRIPTION
**Pull request by alzimmermsft**
This PR prepares the Core libraries for their January 2021 release.

### CODE REVIEW

File: sdk/communication/azure-communication-administration/pom.xml [hunk 1]
```diff
@@ -97,7 +97,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;current} -->
+      <version>1.5.2</version> <!-- {x-version-update;com.azure:azure-core-test;current} -->
```
Comment (alzimmermsft) [inline sdk/communication/azure-communication-administration/pom.xml:100]: This should be using a dependency tag

File: sdk/core/azure-core-amqp/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 2.1.0-beta.1 (Unreleased)
+## 2.0.1 (2021-01-11)
+
+### New Features
+
+- Changed connections from sharing the global `Schedulers.single()` to having a `Scheduler.newSingle()` per connection.
```
Comment (alzimmermsft) [inline sdk/core/azure-core-amqp/CHANGELOG.md:7]: @conniey @srnagar @hemanttanwar

File: sdk/core/azure-core-http-jdk-httpclient/pom.xml [hunk 3]
```diff
@@ -62,27 +62,27 @@
...
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.6.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-core-test;current} -->
+      <version>1.5.2</version>  <!-- {x-version-update;com.azure:azure-core-test;current} -->
       <scope>test</scope>
     </dependency>
```
Comment (alzimmermsft) [inline sdk/core/azure-core-http-jdk-httpclient/pom.xml:81]: Dependency listed twice

File: sdk/resourcemanager/azure-resourcemanager-resources/pom.xml [hunk 4]
```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-management</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
```
Comment (alzimmermsft) [inline sdk/resourcemanager/azure-resourcemanager-resources/pom.xml:60]: This should be using the dependency tag

File: sdk/resourcemanager/azure-resourcemanager-test/pom.xml [hunk 5]
```diff
@@ -62,7 +62,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-management</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
```
Comment (alzimmermsft) [inline sdk/resourcemanager/azure-resourcemanager-test/pom.xml:65]: This should be using the dependency tag

File: sdk/communication/azure-communication-administration/pom.xml (hunk 1 above)
Comment (samvaity) [reply sdk/communication/azure-communication-administration/pom.xml:100]: @ankitarorabit ^^

**Review by samvaity — APPROVED** [review]

File: sdk/core/azure-core-amqp/CHANGELOG.md (hunk 2 above)
Comment (hemanttanwar) [reply sdk/core/azure-core-amqp/CHANGELOG.md:7]: per connection ->  `per connection to **improve performance`**

**Review by hemanttanwar — APPROVED** [review]
LGTM

File: sdk/resourcemanager/azure-resourcemanager-resources/pom.xml (hunk 4 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-resources/pom.xml:60]: Currently mgmt SDK code has a dependency on unreleased azure-core-management. After azure-core-management got released we will revert it back to dependency tag.

Comment (samvaity) [reply sdk/resourcemanager/azure-resourcemanager-resources/pom.xml:60]: It is recommended to use the `unreleased` version tag in these cases.
As mentioned [here](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#current-version-dependency-version-unreleased-dependency-version-and-released-beta-dependency-version),

>Unreleased Dependency version – Whenever possible, libraries should be using the latest released version for dependencies but there is the case where active development in one library is going to be needed by another library or libraries that are built in separate pipelines.

An unreleased entry needs to be created in version_client.txt, under the unreleased section, with the following format: `unreleased_<groupId>:<artifactId>;dependency-version`.

The dependency update tags in the pom files that required this dependency would now reference `{x-version-update;unreleased_com.azure:azure-core;dependency}`.

**Pull request merged**